miniapps/common/mesh_extras.cpp

```diff
@@ -234,6 +234,52 @@ void AttrToMarker(int max_attr, const Array<int> &attrs, Array<int> &marker)
    }
 }
 
+void KershawTransformation::Eval(Vector &V, ElementTransformation &T,
+                                 const IntegrationPoint &ip)
+{
+   V = 0.0;
+   Vector pos(dim);
```

Comment:
It is assumed that `dim>=2`, this should be confirmed with an `MFEM_VERIFY` or something similar.

Reply:
Yes, that check is in the constructor.

Reply:
I missed that. And that's a much better place for it. Thanks!